sources/PRETTY

```diff
@@ -1,16 +1,16 @@
 (DEFINE-FILE-INFO PACKAGE "INTERLISP" READTABLE "INTERLISP" BASE 10)
 
-(FILECREATED " 8-Feb-2023 16:21:26" {DSK}<home>larry>il>medley>sources>PRETTY.;3 65500  
+(FILECREATED " 2-Feb-2024 17:54:23" {DSK}<mnt>e>Interlisp>medley>sources>PRETTY.;2 66897  
 
-      :EDIT-BY "lmm"
+      :EDIT-BY "mth"
 
-      :CHANGES-TO (FNS PRINTDATE1)
+      :CHANGES-TO (FNS PRINTCOPYRIGHT1 PRINTCOPYRIGHT)
 
-      :PREVIOUS-DATE "19-Jan-2022 20:35:18" {DSK}<home>larry>il>medley>sources>PRETTY.;1)
+      :PREVIOUS-DATE " 8-Feb-2023 16:21:26" {DSK}<mnt>e>Interlisp>medley>sources>PRETTY.;1)
 
 
 (* ; "
-Copyright (c) 1984-1990, 1999, 2018, 2023 by Venue & Xerox Corporation.
+Copyright (c) 1984-1990, 1999, 2018, 2023-2024 by Venue & Xerox Corporation.
```

Comment:
But the copyright owners were something like 1984-1990 Xerox 1999 Venue 2018 Ron Kaplan 2023-24 Interlisp.org.  Rather than making auto-generation of (legally dubious) copyright and license statements to make it simpler. Oh, like a file property you can set to point people to the LICENSE statement.